app/routes.js

```diff
@@ -124,7 +129,6 @@ export const routes = (authProvider) => {
     const { userId, organisationId } = req.params;
     const accessToken = extractAccessToken({ req, tokenType: 'access' });
     await postUserStatus({ userId, accessToken, status: 'enable' });
-    logger.info(`navigating to enable user: ${userId} confirmation page`);
     res.redirect(`/organisations/${organisationId}/${userId}/enable`);
```

Comment:
why have we removed the loggers? I thought they were useful

Reply:
because it logs inside the route that it is redirected to so it logs the same thing twice